app/views/shopping_lists/index.html.haml

```diff
@@ -0,0 +1,19 @@
+.d-grid.gap-2.d-md-block.mx-5
+  %button.btn.btn-dark.me-2.position-relative(type="button")= link_to 'New Shopping List', new_shopping_list_path, class: 'text-light text-decoration-none stretched-link'
+
+#shopping-lists-container.d-flex.my-2.mx-5.bg-light.position-relative
+  %nav#shopping-lists-index-navigation.d-flex.flex-column.align-items-start.p-2
+    - @shopping_lists.group_by{|li| li.status}.sort.reverse.each do |status, lists|
```

Comment:
You can use [& shortcut](https://dev.to/junko911/ruby-iteration-shortcut-with-ampersand-operator-923) `group_by(&:status)`